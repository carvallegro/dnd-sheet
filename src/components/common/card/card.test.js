import React from 'react'
import { shallow } from 'enzyme'

import Card from './index'

describe('<Card />', () => {
  it('renders', () => {
    const wrapper = shallow(<Card />)
    expect(wrapper).toMatchSnapshot()
  })

  it('renders with title', () => {
    const wrapper = shallow(<Card title="test title" />)
    expect(wrapper).toMatchSnapshot()
  })
})
