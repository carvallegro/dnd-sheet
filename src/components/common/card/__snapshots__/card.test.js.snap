// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Card /> renders 1`] = `<styled.div />`;

exports[`<Card /> renders with title 1`] = `
<styled.div>
  <styled.p>
    test title
  </styled.p>
</styled.div>
`;
